src/store/index.js

```diff
@@ -5,13 +5,14 @@ import getters from './getters'
 Vue.use(Vuex)
 
 // https://webpack.js.org/guides/dependency-management/#requirecontext
-const modulesFiles = require.context('./modules', false, /\.js$/)
+const modulesFiles = require.context('./modules', true, /\.js$/)
 
 // you do not need `import app from './modules/app'`
 // it will auto require all vuex module from modules file
 const modules = modulesFiles.keys().reduce((modules, modulePath) => {
   // set './app.js' => 'app'
-  const moduleName = modulePath.replace(/^\.\/(.*)\.\w+$/, '$1')
+  var moduleName = modulePath.replace(/^\.\/(.*)\.\w+$/, '$1')
+  moduleName = moduleName.substring(moduleName.indexOf('/') + 1)
```

Comment:
Is it better to keep the directory structure?

`ADempiere/browser` vs `browser`

Reply:
Is better if you want to implement this great functionality for a specific application, here exist a implementation of vue-element-admin for ADempiere: https://github.com/erpcya/adempiere-vue

With it is easy get changes from your project for any fork and implement specific functionality without overwrite default behavior


Best regard